WebView/app/src/main/assets/main.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function sendAndroidMessage() {
+	Weather.sendMessage("The weather in WebView City today is: " + document.getElementById("description").innerHTML);
```

Comment:
1. Can we avoid hardcoding the location name here?
2. Can we limit line length to 80 columns? https://google.github.io/styleguide/jsguide.html#formatting-column-limit
3. Would `innerText` be better here? We don't want to send HTML tags to Android.